### ISSUE TITLE
[Bug]: When inserting zero column data, errors occur.

### ISSUE THREAD

**Issue by chrysanthemum-boy** [infiniflow/infinity#687]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Branch name

main

### Commit ID

6f8b3cb2

### Actual behavior

When inserting zero column data, errors occur.
```
[10:53:55.171] [1094831] [critical] Error: SEGMENT FAULTS
[10:53:55.171] [1094831] [critical] 0, ./build/src/infinity(+0x50cd89) [0x562347ebdd89]
[10:53:55.171] [1094831] [critical] 1, ./build/src/infinity(+0x106401) [0x562347ab7401]
[10:53:55.171] [1094831] [critical] 2, /lib/x86_64-linux-gnu/libc.so.6(+0x42990) [0x7f6205442990]
[10:53:55.171] [1094831] [critical] 3, ./build/src/infinity(+0x66edc3) [0x56234801fdc3]
[10:53:55.171] [1094831] [critical] 4, ./build/src/infinity(+0x27a3a4) [0x562347c2b3a4]
[10:53:55.171] [1094831] [critical] 5, ./build/src/infinity(+0x27c1cf) [0x562347c2d1cf]
[10:53:55.171] [1094831] [critical] 6, ./build/src/infinity(+0xcfca03) [0x5623486ada03]
[10:53:55.171] [1094831] [critical] 7, /lib/x86_64-linux-gnu/libc.so.6(+0x97b5a) [0x7f6205497b5a]
[10:53:55.171] [1094831] [critical] 8, /lib/x86_64-linux-gnu/libc.so.6(+0x1285fc) [0x7f62055285fc]
terminate called without an active exception
Aborted

```

### Steps to reproduce

```Markdown
Function test code can be found in python/test/test_insert.py`test_insert_zero_column`
```

**Comment by JinHai-CN** [comment]
Fixed by #700

**Comment by chrysanthemum-boy** [comment]
It looks fixed.
